src/inferenceql/query/tuple.cljc

```diff
@@ -24,7 +24,7 @@
 (defn select-attrs
   "Retrives tuple `tup` with only the attributes in `attrs`."
   [tup attrs]
-  (let [names (into #{} (map name) attrs)]
+  (let [names (into #{} (map name) attrs)]                  ; FIXME: should this be clojure.core/name?
```

Comment:
let's get this one resolved. Reviewing this sort of PR is where I miss types, I can't see what `attrs` should be...

Reply:
I'm 99% certain it's a vec of keys (which should be strings these days), based on the attribute code I see elsewhere. In that case, it should almost certainly be `clojure.core/name` here.

I think the reason nobody's noticed the error is because the `select-attrs` fn is only called in `i.q.relation/project`, which is never used.

Reply:
Yep, that's it:

```
(-> (strict/q "SELECT * FROM tbl LIMIT 1" db)
    first
    (tuple/select-attrs #{"Launch_Site" "Contractor"}))
=> {}

(defn select-attrs
  "Retrives tuple `tup` with only the attributes in `attrs`."
  [tup attrs]
  (let [names (into #{} (map clojure.core/name) attrs)]               
    (medley/filter-keys (comp names clojure.core/name)
                        tup)))
=> #'inferenceql.query.tuple/select-attrs

(-> (strict/q "SELECT * FROM tbl LIMIT 1" db)
    first
    (tuple/select-attrs #{"Launch_Site" "Contractor"}))
=> {"Launch_Site" "Cape Canaveral", "Contractor" "Lockheed Martin"}
```

Reply:
Fixed.